### DISCUSSION TITLE
How to have a multiple search? Dropdown or search module?

### DISCUSSION THREAD

**Discussion by smartm0use** [fomantic/Fomantic-UI#1857] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)]
Hello, I need a multiple search selection that make an async search to the server.

Which between dropdown and search best suit my needs?
Can I have async search functionality in the dropdown with multiple selections?
Or, can I have multiple selections in the search module?

**Comment by smartm0use** [comment]
Currently isn't this possible in Fomantic-UI?

**Comment by lubber-de** [comment]
Multiple Selections are only available in the dropdown  module. 

If you want to do async calls from the given selections, then you should trigger the call from either a separate button or use the dropdowns `onChange`, `onAdd`, `onRemove` callbacks.
